src/components/HeaderNavMenu.jsx

```diff
@@ -30,8 +52,27 @@ function HeaderNavMenu() {
           </MuiLink>
         </Grid>
       ))}
-      <Grid item>
-        <LoginButton className={classes.menuItem} />
+      <Grid item className={classes.avatarMenu}>
+        {(loading && <null />) ||
+          (isSignedIn ? (
```

Comment:
This whole `isSignedIn ? A : B` thing can surrounded by a [Zoom transition](https://material-ui.com/components/transitions/#zoom):
```jsx
        (<Zoom in>
          {isSignedIn ? A :B }
        </Zoom>)
```

The code is only ever reached if `!loading` so `in` is always `true`.

Reply:
Ok yep that worked